ui/app/rif/pages/domainsDetailPage/domainDetailActive/domainDetailActive.js

```diff
@@ -158,100 +160,156 @@ class DomainsDetailActiveScreen extends Component {
     return rns.contracts.publicResolver;
   }
 
+  convertChainAddressesToTableData () {
+    const data = [];
+    this.state.chainAddresses.map((chainAddress, index) => {
+      const address = getChainAddressByChainAddress(chainAddress.chain);
+      const icon = address.icon ? address.icon : DEFAULT_ICON;
+      const tableRow = {};
+      tableRow.content =
+        <div className={''}>
+          <div className={''}>
+            <FontAwesomeIcon icon={icon.icon} color={icon.color} className={''}/>
+            <span>{address.symbol}</span>
+          </div>
+          <div className={''}>
+            <span>{chainAddress.address}</span>
+          </div>
+        </div>
+      tableRow.actions =
+        <div className={''}>
+          <FontAwesomeIcon
+            icon={faPen}
+            // color={'#FFFFFFFFFFF'}
```

Comment:
Remove if unused

Reply:
I was going to let this color att here, so @fmelo can use this. But he can also assign the color in CSS